spec/controllers/searches/users_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Searches::UsersController, type: :controller do
+  let(:user) { create(:user) }
+  let(:profile) { create(:profile, user_id: user.id) }
```

Comment:
`let` はテストする過程で`login_user(user)`みたいに参照しないと実行されないので、`profile ` をテスト内で参照しないとデータが作られません。
この場合`let!`にするか`before`ブロックで囲めばテスト実行前に必ず実行されます。

```suggestion
  before { create(:profile, user_id: user.id) }
```

Reply:
ご指摘ありがとうございます。let, let!, beforeへの理解不足でした。
書き直してpushします。